roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -126,7 +126,7 @@ rgw frontends = {{ frontend_line(radosgw_frontend_type) }} {{ radosgw_frontend_o
 {% if 'num_threads' not in radosgw_frontend_options %}
 rgw thread pool size = {{ radosgw_thread_pool_size }}
 {% endif %}
-{% if rgw_multisite | bool %}
+{% if hostvars[host]['rgw_multisite'] | bool %}
```

Comment:
This won't work because when using the default value (ie: not configuring the variable in group_vars) so you probably need to add the `default` filter with the `rgw_multisite` value.